docs/server/configuration/security.md

```diff
@@ -722,32 +722,87 @@ programmatically.
 This allows administrators to define stream access policies for EventStoreDB based on stream prefix.
 
-### Configuration
+### Enabling
 
 You require a [license key](../quick-start/installation.md#license-keys) to use this feature.
 
-Refer to the [configuration guide](../configuration/README.md) for configuration mechanisms other than YAML.
+The stream access policy is configured by the last event in the `$authorization-policy-settings` stream.
 
-The feature can be enabled by setting the `Authorization:PolicyType` option to `streampolicy`.
+Enable stream policies by appending an event to the `$authorization-policy-settings` stream:
 
-You can do this by adding the following to the server configuration file:
+```json
+{
+    "streamAccessPolicyType": "streampolicy"
+}
+```
 
-```yaml
-Authorization:
-  PolicyType: streampolicy
+And disable stream policies by setting the stream access policy type back to `acl`:
+
+```json
+{
+    "streamAccessPolicyType": "acl"
+}
 ```
 
+When stream policies are enabled, you should see the following logs:
+
 You can check that the feature is enabled by searching for the following log at startup:
 
 ```
-[20056, 1,11:14:40.088,INF] ClusterVNodeHostedService Loaded authorization policy plugin: StreamPolicyPlugin version 24.10.0.0 (Command Line: streampolicy)
-[20056, 1,11:14:40.088,INF] ClusterVNodeHostedService Using authorization policy plugin: StreamPolicyPlugin version 24.10.0.0
+[22860,28,17:30:42.247,INF] StreamBasedAuthorizationPolicyRegistry New Authorization Policy Settings event received
+[22860,28,17:30:42.249,INF] StreamBasedAuthorizationPolicyRegistry Starting factory streampolicy
+...
+[22860,28,17:30:42.264,INF] StreamBasedPolicySelector      Existing policy found in $policies stream (0)
+[22860,28,17:30:42.269,INF] StreamBasedPolicySelector      Successfully applied policy
+[22860,28,17:30:42.271,INF] StreamBasedPolicySelector      Subscribing to $policies at 0
+```
+
+If you do not have a valid license, stream policies will log the following error and will not start. EventStoreDB will continue to run with the current policy type:
+
+```
+[22928, 1,17:31:56.879,INF] StreamPolicySelector           Stream Policies plugin is not licensed, stream policy authorization may not be enabled.
+...
+[22928,28,17:31:57.030,ERR] StreamPolicySelector           Stream Policies plugin is not licensed, cannot enable Stream policies
+[22928,28,17:31:57.030,ERR] StreamBasedAuthorizationPolicyRegistry Failed to enable policy selector plugin streampolicy. Authorization settings will not be applied
 ```
 
 ::: note
 If you enable the Stream Policy feature, EventStoreDB will not enforce [stream ACLs](#access-control-lists).
 :::
 
+### Overriding the default
+
+If the `$authorization-policy-settings` stream is empty and there is no configuration, EventStoreDB will default to using ACLs.
+
+If you would rather default to stream policies, you can do this by setting the `Authorization:PolicyType` option to `streampolicy`.
```

Comment:
it might be worth mentioning what happens when both the `$authorization-policy-settings` stream and `PolicyType` is specified is present in the server configuration file (it appears that the stream takes precedence)

Reply:
I've added a note and renamed the `Authorization:PolicyType` setting to `Authorization:DefaultPolicyType`